up42/utils.py

```diff
@@ -388,7 +387,7 @@ def replace_page_query(url: str, new_page: int) -> str:
     """
     parsed_url = urlparse(url)
     query_params = parse_qs(parsed_url.query)
-    query_params["page"] = str(new_page)  # type: ignore
+    query_params["page"] = [str(new_page)]
```

Comment:
Why do the change type here?

Reply:
Otherwise mypy throws an error:
 `error: Incompatible types in assignment (expression has type "str", target has type "list[str]")  [assignment]`
 
 If, I am not mistaken, this requirement is coming from the `parse_qs` function.
 
<img width="494" alt="image" src="https://github.com/up42/up42-py/assets/54374009/6cec83fa-7365-4f8c-b96c-18efd491c6e6">